### PR TITLE
StatusBar implemented (closes #479)

### LINKED ISSUE #479: Redesign test run UI
Implement status bar at the bottom of the page which will replace `Loading page...` screen and waiting element modal popup and add additional test run info. Status bar will become nearly completely opaque on hover (if you want to observer the page below it):
image

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin @inikulin

### CODE REVIEW

File: src/client/ui/status-bar/index.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+    _disposeStatusBar () {
+        var statusBarHeight = styleUtils.getHeight(this.statusBar);
+        var top             = styleUtils.getHeight(window) - statusBarHeight;
+
+        styleUtils.set(this.statusBar, 'top', top + 'px');
+    }
+
+    _setSizeStyle (windowWidth) {
+        if (windowWidth > MIDDLE_WINDOW_WIDTH) {
```
Comment (inikulin) [inline src/client/ui/status-bar/index.js]: Can't we do this via CSS `@media` query?

File: src/client/ui/status-bar/progress-bar/determinate-behavior.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+var shadowUI      = hammerhead.shadowUI;
+var nativeMethods = hammerhead.nativeMethods;
+var styleUtils    = testCafeCore.styleUtils;
+
+
+const DETERMINATE_STYLE_CLASS   = 'determinate';
+const ANIMATION_UPDATE_INTERVAL = 10;
+
+
+export default class DeterminateBehavior {
```
Comment (inikulin) [inline src/client/ui/status-bar/progress-bar/determinate-behavior.js]: `DeterminateIndicator` and `IndeterminateIndicator`. `DeterminateBehavior` is ambigious

**Comment by inikulin** [comment]
\r-

File: src/client/ui/status-bar/index.js (hunk 1 above)
Comment (helen-dikareva) [inline src/client/ui/status-bar/index.js]: Not, it doesnot work in all IEs

Comment (inikulin) [inline src/client/ui/status-bar/index.js]: OK, then

**Comment by helen-dikareva** [comment]
I'll fix tests

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
rebase required

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
yahoo

**Pull request merged**